### PR TITLE
add string-literal.jsh

### PR DESCRIPTION
**Pull request by wangkang09**
https://github.com/wangkang09/java-up

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -21,6 +21,6 @@
 *.zip
 *.tar.gz
 *.rar
-
+.idea
```
Comment (XueleiFan) [inline .gitignore:24]: Please don't ignore the .idea directory, which could be useful for other users.   I would suggest removing this update from the pull request.

File: src/main/java/co/ivi/jus/jshell/review/wangkang/string-literal.jsh [hunk 2]
```diff
@@ -0,0 +1,11 @@
+"hello" == "hello";//true
+
+"hello1" == "hello" + "1";//true
+
+"hello1" == "hello" + new String("1");//false，地址不一样
```
Comment (XueleiFan) [inline src/main/java/co/ivi/jus/jshell/review/wangkang/string-literal.jsh:5]: Hm, a nice test!

**Review by XueleiFan — CHANGES_REQUESTED** [review]
Looks good to me, except the update in .gitignore.

File: .gitignore (hunk 1 above)
Comment (wangkang09) [reply .gitignore]: ok, causeof some changes happend in .idea directory when i run the project..

File: src/main/java/co/ivi/jus/jshell/review/wangkang/string-literal.jsh (hunk 2 above)
Comment (wangkang09) [reply src/main/java/co/ivi/jus/jshell/review/wangkang/string-literal.jsh:5]: jshell is realy a nice tool to do some small test!

**Review by XueleiFan — APPROVED** [review]
Looks good to me.  Thanks!

**Pull request merged**